### DISCUSSION TITLE
Is there a schedule for release to npm ?

### DISCUSSION THREAD

**Discussion by rogerhyam** [ProjectMirador/mirador#3473] [Q&A] [answered]
I'd really like to be able to recommend to people to use the npm package delivered over unpkg.com but it doesn't currently support dragging manifests onto it. The master branch does support this so people would have to build their own release to get this feature.

Is there a regular build/release schedule that I could used to estimate when this functionality will be available through the content delivery network or to install with "npm install"?

**Comment by camillevilla** [comment, marked as answer]
hi @rogerhyam. We don't have a formal schedule for cutting releases to npm. I think we can probably cut a new release in the next week or two. Feel free to drop a message in the #mirador channel of the [IIIF community slack](http://bit.ly/iiif-slack) in the future.
